brainscore/metrics/accuracy_delta.py

```diff
@@ -0,0 +1,108 @@
+import itertools
+
+import numpy as np
+
+from brainscore.metrics import Metric, Score
+import random
+from brainio.assemblies import BehavioralAssembly
+
+
+# controls how many half-splits are averaged together to get human delta.
+HUMAN_SPLITS = 100
+
+
+class AccuracyDelta(Metric):
+
+    def __init__(self, image_types: list):
```

Comment:
Wouldn't this be a bit more general?
```suggestion
    def __init__(self, condition_key: str, condition_values: List):
```
e.g. `condition_key=image_type, condition_values=['w', 'f']`

This way you might more easily be able to apply the metric for other benchmarks.

Although if this entire metric is specific to Baker, let's maybe call it `BakerAccuracyDelta`?